(#20) Fix fabulous 0.53 Compatibility

franko-franicevich opened the pull request:
https://github.com/Zaid-Ajaj/fabulous-simple-elements/issues/20

Fabulous 0.53 seems to have (accidentally?) removed the foregroundColor property from Span.

Temporarily removing foregroundColor attribute from Span in simple-elements to match.

COMPATIBILITY ISSUE: This might result in incompatibility with clients of the library if they're using span.foregroundColor. But in that case, they're not going to work against fabulous 0.53 anyway until the property is added back.

Zaid-Ajaj commented:
Thanks! I merged and published version 0.42.0 that is compatible with latest Fabulous 0.53.2 :rocket: it should be available soon on nuget

franko-franicevich commented:
Wonderful!

Tim@fabulous has this to say:
```
Hi @franko-franicevich

We removed it because it has been marked obsolete for some times in Xamarin.Forms, and replaced with TextColor.
In Xamarin.Forms, IntelliSense stopped showing it (it's only there to allow migration from Xamarin.Forms 3.1).
https://github.com/xamarin/Xamarin.Forms/blob/5eec0f5936d8c8a19ae6c9c4c3387b141f3acfe6/Xamarin.Forms.Core/Span.cs#L53-L55

I would recommend updating fabulous-simple-elements instead.```

So it sounds like this is a *permanent* fix, not temporary. (and some of the MS docs need to be updated to reflect the fact foregroundColor is obsolete.)

If you want, I can issue another pull request deleting those (now inaccurate) comments, unless it's just easier for you to do so now that you have the diff.

Zaid-Ajaj commented:
@franko-franicevich I read the comments on the issue filed at the Fabulous repo and removed the comments accordingly since they are no longer needed :smile:

franko-franicevich commented:
Great, thanks again for your quick response on this!
